cmd/osmosisd/cmd/balances_from_state_export.go

```diff
@@ -303,3 +305,75 @@ Example:
 
 	return cmd
 }
+
+// BalancesToCSVCmd generates a airdrop.csv from a provided exported balances.json.
+func StakedToCSVCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "staked-to-csv [input-balances-file] [output-airdrop-csv]",
+		Short: "Export a airdrop csv from a provided balances export",
+		Long: `Export a airdrop csv from a provided balances export (from export-derive-balances)
+Example:
+	osmosisd staked-to-csv ../balances.json ../airdrop.csv
+`,
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			serverCtx := server.GetServerContextFromCmd(cmd)
+			config := serverCtx.Config
+			config.SetRoot(clientCtx.HomeDir)
+
+			balancesFile := args[0]
+
+			snapshotOutput := args[1]
+
+			minStakeAmount, _ := cmd.Flags().GetInt64(FlagMinimumStakeAmount)
+
+			var deriveSnapshot DeriveSnapshot
+
+			sourceFile, err := os.Open(balancesFile)
+			if err != nil {
+				return err
+			}
+			// remember to close the file at the end of the function
+			defer sourceFile.Close()
```

Comment:
nit: consider deferring before err check

Reply:
I get lint error if you error check after defer